backend/Platform/settings/production.py

```diff
@@ -43,3 +43,14 @@
 EMAIL_USE_TLS = True
 
 IS_DEV_LOGIN = os.environ.get("DEV_LOGIN", "False") in ["True", "TRUE", "true"]
+
+# AWS S3
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+AWS_ACCESS_SECRET_ID = os.getenv("AWS_SECRET_ACCESS_KEY")
+AWS_STORAGE_BUCKET_NAME = "penn.platform"
```

Comment:
Also, values such as `AWS_STORAGE_BUCKET_NAME` should be added to vault instead of hardcoded! I just added it, so would be great if you can chaneg it to an `os.getenv` :D